tests/Unit/ProtoTests/BasicOneof/out/src/Request/SupplementaryDataOneof.php

```diff
@@ -67,4 +67,207 @@ class SupplementaryDataOneof
      * lower_camel_case.
      */
     private static $selectedOneofFieldName = '';
+
+    /**
+     * Sets this oneof to extra_description and updates its value.
+     *
+     * @param mixed $extraDescription The new value of this oneof.
```

Comment:
What does `mixed` mean here and should there be a period before the `The` or should that be lower case?

Reply:
Fixed the type, and no for the `The`, as per the rest of our generated docs ([example](https://github.com/googleapis/gapic-generator-php/blob/master/tests/Integration/goldens/asset/src/V1/Gapic/AssetServiceGapicClient.php#L332)).